### ISSUE TITLE
Upload code for TFBS Reward Inference

### ISSUE THREAD

**Issue by HowIII** [yangzhao1230/TACO#3]
Hi, thanks for the great work!

I am wondering if you could upload the code for TFBS reward inference (i.e., mainly the code for training the lightGBM model and use SHAP value for the interpretability analysis)? Thank you!

**Comment by yangzhao1230** [comment]
Hi @HowIII , thank you for your interest!

I assume that you have already extracted the TFBS features and saved them in h5ad format (please refer to regLM's notebooks for this step).

Here I'm providing the lightGBM training code [lightGBM_mbo.py](https://github.com/yangzhao1230/TACO/blob/main/lightGBM_mbo.py) and the TFBS reward inference code https://github.com/yangzhao1230/TACO/blob/main/tfbs_reward_mbo.ipynb for your reference.

**Comment by HowIII** [comment]
Thank you!
